src/ci/resources/papi_v2beta_application.conf

```diff
@@ -77,8 +77,8 @@ backend {
         include "papi_v2beta_provider_config.inc.conf"
         genomics.compute-service-account = "centaur@broad-dsde-cromwell-dev.iam.gserviceaccount.com"
         filesystems.http {}
-        # we don't want to enable this until the file is actually there
-        # reference-disk-localization-manifest-files = ["gs://gcp-public-data--broad-references/refdisk_manifest.json"]
+        # TODO: change this to proper path to reference bucket once manifest is uploaded there
+        reference-disk-localization-manifest-files = ["gs://cloud-cromwell-dev-self-cleaning/refdisk_manifest.json"]
```

Comment:
You might want to stage this under `gs://cloud-cromwell-dev` instead to avoid an unwanted cleaning...

Reply:
Now staged in the references bucket, so let's point at that one if we can.